components/CountryNavigator.js

```diff
@@ -0,0 +1,248 @@
+              <div className="flex gap-2 ">
+                <h3 aria-hidden>
+                  {" "}
+                  {year}{" "}
+                  <span className="text-slate-400">{` | ${nth(
+                    currentCountryData.find((d) => d.year === year)?.ranking
+                  )}`}</span>
+                </h3>
+                <h3></h3>
+              </div>
+            ) : null}
+          </div>
+        ))}
+      </div>
+      <div className={`flex mt-10 gap-4 visible`} ref={flagsContainer}>
+        {lastYear.map((year) => (
+          <div key={`${countries[currentCountryIndex]}-${year}`}>
+            <motion.div key="flag">
+              <Box
+                sx={{
+                  width: flagDimensions.width,
+                  height: flagDimensions.height,
+                }}
+              >
+                <Flag
+                  country={countries[currentCountryIndex]}
+                  year={year}
+                  isTimeline={"false"}
+                  orientation={"horizontal"}
+                  isLastYear={"true"}
+                  isInteractive={false}
+                />
+              </Box>
+            </motion.div>
+          </div>
+        ))}
+        <div style={{ display: "inline-block", alignSelf: "flex-end" }}>
+          <p style={{ color: colorScale("equality") }}>
+            Equality and non-descrimination{" "}
+            {format(".0%")(data[currentCountryIndex].equality)}
+          </p>
+          <p style={{ color: colorScale("family") }}>
+            Family {format(".0%")(data[currentCountryIndex].family)}
+          </p>
+          <p style={{ color: colorScale("hate") }}>
+            Hate crime and hate speech{" "}
+            {format(".0%")(data[currentCountryIndex].hate)}
+          </p>
+          <p style={{ color: colorScale("gender") }}>
+            Legal gender recognition & Intersex bodily autonomy{" "}
+            {format(".0%")(data[currentCountryIndex].gender)}
+          </p>
+
+          <p style={{ color: colorScale("civil") }}>
+            Civil society space {format(".0%")(data[currentCountryIndex].civil)}
+          </p>
+
+          <p style={{ color: colorScale("asylum") }}>
+            Asylum {format(".0%")(data[currentCountryIndex].asylum)}
+          </p>
+        </div>
+      </div>
+      <div
+        className="flex mt-4 text-4xl font-medium gap-14"
+        style={{ width: flagDimensions.width, justifyContent: "space-between" }}
+      >
+        <h2 tabIndex={0}>
+          {countries[currentCountryIndex]}{" "}
+          <span className="text-slate-400">{` | ${nth(
+            currentCountryData.find((d) => d.year === year)?.ranking
+          )}`}</span>
+        </h2>
+        <div className="flex gap-2 text-slate-400">
+          <SortButton
+            onClick={handlePreviousCountry}
+            variant="outlined"
+            size="small"
+          >
+            <ArrowCircleLeftOutlinedIcon></ArrowCircleLeftOutlinedIcon>
```

Comment:
I like these controls! 😄 

Maybe we can add some aria-labels to the icons?

```suggestion
            <ArrowCircleLeftOutlinedIcon aria-label="Next country"/>
```